### PR TITLE
Java Environment Variable Bindings

### PR DESCRIPTION
**Pull request by nrgraham23**
Added the bindings for MPI_GET_VERSION and MPI_GET_LIBRARY_VERSION.

Signed-off-by: Nathaniel Graham <ngraham@lanl.gov>

@hppritcha  @jsquyres

### CODE REVIEW

File: ompi/mpi/java/java/Version.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+package mpi;
+
+/**
+ * Version and Subversion for MPI
+ */
+public final class Version
+{
+private final int version;
+private final int subVersion;
```
Comment (jsquyres) [inline ompi/mpi/java/java/Version.java]: Minor comment: Seems like weird indenting...?

Comment (nrgraham23) [inline ompi/mpi/java/java/Version.java]: It is weird indenting, but I believe it follows the indenting of the rest of the java files.  If you like I can easily change it.

Comment (nrgraham23) [inline ompi/mpi/java/java/Version.java]: I could also fix the indenting for all of the other Java files if you
like.  Its very very simple in eclipse.

Comment (jsquyres) [inline ompi/mpi/java/java/Version.java]: What's the "Normal" Java way?  To my C-trained eye, this looks weird.  But if it's The Way Java People Do It, it's fine.

If it's not the normal way java people do it, then you might as well keep the indenting here in this PR and make a separate PR for just all the whitespace/indenting changes.  Cool?

Comment (nrgraham23) [inline ompi/mpi/java/java/Version.java]: This is not the normal way.  The normal way indents everything inside the class as well.  I will fix the indentation in a separate PR.

**Comment by hppritcha** [comment]
Looks like mlnx jenkins is getting timeouts again on some of its multi-threaded bw tests.

**Comment by hppritcha** [comment]
:+1:
